modules/determining-where-s2i-issues-occur.adoc

```diff
@@ -0,0 +1,10 @@
+// Module included in the following assemblies:
+//
+// * support/troubleshooting-s2i.adoc
+
+[id="determining-where-s2i-issues-occur_{context}"]
+= Determining where issues occur in the Source-to-Image process
+
+Source-to-Image (S2I) is a tool for building reproducible, Docker-formatted container images. It produces ready-to-run images by injecting application source into a container image and assembling a new image. The new image incorporates the base image (the builder) and built source. S2I also builds a Deployment Configuration which is triggered after an S2I image build succeeds. This deploys application containers based on the new image, as well as required cluster resources.
```

Comment:
Check capitalization on "Deployment Configuration". Do you mean "Deployment configuration", or do you mean a "DeploymentConfig" ?

Reply:
I have changed this to 'deployment configuration'. @bergerhoffer should deployment be capitalised?